### PR TITLE
Add New GitHub Action for SLO Tests Integration with SDK

### PR DESCRIPTION
**Pull request by polRk**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Migrate to new slo action

## What is the current behavior?

Issue Number: N/A

### CODE REVIEW

**Comment by polRk** [comment]
# SLO Report 

### Operation Success Rate

```mermaid
---
config:
    xyChart:
        width: 1200
        height: 400
    themeVariables:
        xyChart:
            titleColor: "#222"
            backgroundColor: "#fff"
            xAxisLineColor: "#222"
            yAxisLineColor: "#222"
            plotColorPalette: "#FF7F0E,#1F77B4,#D62728,#2CA02C,#9467BD,#8C564B,#E377C2,#7F7F7F,#BCBD22,#17BECF"
---
xychart-beta
    title "operation_type=read"
    x-axis "Time, m" 0 --> 10
    y-axis "Success Rate, %" 90 --> 111
    line [100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100]
```


```mermaid
---
config:
    xyChart:
        width: 1200
        height: 400
    themeVariables:
        xyChart:
            titleColor: "#222"
            backgroundColor: "#fff"
            xAxisLineColor: "#222"
            yAxisLineColor: "#222"
            plotColorPalette: "#FF7F0E,#1F77B4,#D62728,#2CA02C,#9467BD,#8C564B,#E377C2,#7F7F7F,#BCBD22,#17BECF"
---
xychart-beta
    title "operation_type=write"
    x-axis "	Time, m" 0 --> 10
    y-axis "Success Rate, %" 89 --> 111
    line [100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100,100]
```


### Operations Per Second

```mermaid
---
config:
    xyChart:
        width: 1200
        height: 400
    themeVariables:
        xyChart:
            titleColor: "#222"
            backgroundColor: "#fff"
            xAxisLineColor: "#222"
            yAxisLineColor: "#222"
            plotColorPalette: "#FF7F0E,#1F77B4,#D62728,#2CA02C,#9467BD,#8C564B,#E377C2,#7F7F7F,#BCBD22,#17BECF"
---
xychart-beta
    title "operation_type=read"
    x-axis "Time, m" 0 --> 10
    y-axis "Operations" 85 --> 895
    line [812.786,807.357,807.786,802.571,803.5,805.857,806.429,805.214,804.286,803.071,798.357,796,796.071,804,803.214,807.929,807.643,807.5,806.214,801.357,801.357,797.143,796.857,802.857,804.929,805.571,807.857,804.857,803.143,802.143,804.214,800.286,800.357,803.071,806.5,809.5,808.857,805.857,801.5,801.857,794.429,790.143,795.929,796.5,800.357,804.929,804,805.071,802.643,801.643,801.357,798.786,800.071,800.357,803.786,811,806.429,806.214,801.143,798.5,798.214,798.571,798,798.571,800.429,800.643,802.214,804.286,804.071,797,802.357,798.286,801.571,803.143,803.929,801.643,800.786,804.5,804.571,804.714,803.714,798.429,799.214,802.857,802,807.5,803.071,801.143,801.143,801.071,806.857,801.071,800.571,801.929,801.286,804.143,803.143,798.786,798.071,798.571,805.357,804.5,805.571,805.571,799.357,802.214,802,801.143,806.071,810.643,811.429,811.357,808.071,807.429,801.286,809.143,809.286,807.857,811,810.643,808.857,810.714,805.357,798.357,796.643,798.214,797.5,795.714,799.214,796.643,796.929,799.286,799.214,798.786,801.071,801,801.929,804,805.786,804.714,804.5,806.786,802.143,800.571,802.214,801.286,804.714,803.071,799.857,798.143,797.214,798.143,796.071,798.571,802.786,800.571,802.929,801,798.214,800.5,795.786,797.714,802.857,805,805.786,806.143,807.071,806.786,804.571,804.786,803.571,811.286,810.643,806.286,805.5,805.214,801.714,800.143,807.714,806.429,806.214,804.5,800.643,800.143,805.5,799.5,800.429,802.857,804.357,807,807.929,807.786,800.571,800.857,801.643,802.714,806.286,807.143,804.929,804.786,804.071,804.429,799,795.214,793.857,792.214,792.071,793.143,792.429,790.071,787.143,786.643,780,785.286,785.857,784.5,788.214,788.571,788.214,789.643,787.857,786.643,789.643,789.786,792.857,792.071,795.214,787,787.214,789.357,787.429,792,790.429,790.071,789.357,789.143,787.857,788.714,787.286,789.429,789.429,788.643,786.857,785.5,790.571,787,788.071,789.786,792.143,791.5,786.286,785.786,782.5,779.786,778.143,784.857,786.714,787.214,784.214,783.643,783.714,784.857,782.286,788.714,792,790.429,791.786,792.143,793.929,788.214,791.786,793,790.143,788.071,789,787.929,788.5,781.143,780.714,782.429,788.286,789.5,788.214,789.143,783.214,782.571,783.786,783,784.429,781.571,781.714,780.857,781.143,779.5,772.286,770.714,770.214,770.643,772.429,774.929,774.571,773.5,772.143,771.286,771.214,773.786,779.286,781.786,781.929,783.929,785.214,786,791.643,787.071,787.929,791,792.143,797.5,799.857,799.786,796.643,795.643,797.071,796.357,798.071,795.643,796.286,796.857,796,794.643,791.286,793.857,791.786,792.286,790.429,790.714,790.571,791.571,791.286,793.214,786.429,785.5,785.857,784.643,789.5,782.571,783.286,788.857,788.214,787.143,787.5,786.286,784.286,781.214,783.357,785.643,788.786,791.714,784.929,786.929,785,772.5,773.643,769.929,768.429,773.429,774,775.071,771.571,769.143,766.571,769.357,771.143,774.214,774.714,779.357,777.571,780.286,780.071,777.214,777.143,777.5,782.071,782.071,781.786,780.786,782.857,778.571,779.857,779.857,780.214,782.786,781.143,779.571,781.714,781.929,776.286,778.929,780.071,776.357,783.071,784.429,782.786,782.5,781.214,781.857,785.286,784.214,784.714,787.286,788.429,785.214,784.786,783.5,776.214,774.071,774.714,779.5,781.857,781.357,787.429,784.857,780.571,778.357,779.929,785.429,784.571,791.857,790.357,791.071,791.5,787.857,784.429,779.786,774.071,775.929,776.429,783.5,782.714,777.571,777.286,772,773.429,775.286,775.857,778.786,778.714,783.071,784,789.357,789.714,780.429,779.214,778.286,778.786,779.429,778.286,775.643,774.071,779,782.571,782.143,781.143,778.929,781.357,784.571,791.929,793,793.286,794.786,796.643,797.714,799.357,794.857,794.5,795.857,795.571,794.714,794,794.429,788.929,789.571,792.143,789.643,788.5,797.571,802.357,801.071,801.714,801.571,800.857,797.286,797.714,797.714,795.429,796.5,796.286,800.714,801.714,794.571,790.929,790.429,788.571,786.143,785.857,791.5,789.357,789.286,796.714,797.357,793.714,789.143,787.214,786.143,786.429,785.429,786.571,792.857,793.214,788.714,788.286,785.857,778.429,775.143,773.214,774.071,775.5,774.429,779.286,779.214,777.714,774,773.429,773.643,774.286,773.714,774.857,776.786,778.929,780.571,785.714,785.286,775.429,775.929,774.429,772.571,773.929,774.071,780.286,780.786,781,780.857,780.5,783.286,776.714,779.429,783.429,789.5,793.071,794.071,794,797.786,794.5,799.929,799.929,798.714,799.214,795.643,798.571,797.286,797.143,794.071,793.357,731.415,679.621,624.903,570.738,514.952,463.365,413.216,361.634,309.028,255.809,203.256,151.098,95.073]
```


```mermaid
---
config:
    xyChart:
        width: 1200
        height: 400
    themeVariables:
        xyChart:
            titleColor: "#222"
            backgroundColor: "#fff"
            xAxisLineColor: "#222"
            yAxisLineColor: "#222"
            plotColorPalette: "#FF7F0E,#1F77B4,#D62728,#2CA02C,#9467BD,#8C564B,#E377C2,#7F7F7F,#BCBD22,#17BECF"
---
xychart-beta
    title "operation_type=write"
    x-axis "Time, m" 0 --> 10
    y-axis "Operations" 11 --> 106
    line [79,79.143,78.929,85.214,85.143,86.286,84.857,85.929,86.357,86.5,91.143,91.643,92.857,86.214,86.286,86.143,86.357,87.143,88.714,88.643,91,92.286,91.286,85.929,85.929,86,79,86,85.929,85.929,85.929,85.929,85.429,85.143,79.571,79.786,80.786,86.143,86.143,86.286,93.286,90.857,86.357,86.214,80.786,80.714,80.714,80.786,85.714,86.143,86.214,86.214,86.357,86.286,86.214,81.643,86.143,86.286,91.714,91.786,92.286,85.571,86.214,86.214,86.071,86.071,85.929,86,86.071,93.071,86.143,86.143,86.143,86.143,86.143,93.071,93.071,87.571,86.143,86.143,86.143,86.143,86.143,83.357,86.071,86.143,86.143,86.143,86.143,84.786,79.143,84.714,86.143,86.143,86.143,86.143,86.143,88.929,88.786,86.286,79.214,79.214,79.214,80.571,86.214,86.143,83.786,83.714,79.143,72.071,72.071,72.071,76.429,78.857,85.929,79,79,79,79,79,81.357,74.429,79.071,86.143,86.143,86.143,86.143,86.143,84.714,87.786,86.286,86.286,79.643,79.786,79.714,86.357,86.214,86.214,85.929,85.071,84.786,85,85.143,87.071,86.143,86.071,85.714,85.571,85.643,85.929,85.857,85.571,84.857,83.857,80.571,80.214,81.643,83.571,86,82.429,86.071,86,78.929,77.929,77.714,77.714,78.286,78.5,79.429,79.143,78.857,71.857,71.857,75.5,78.857,78.929,86,87.071,80.429,80.786,81.214,82.857,85.5,84.286,79.143,86,86.143,86.143,84.714,79.643,79.143,79.143,83.643,83.143,82.643,80.714,79.143,80.714,82.857,82.857,82.857,82.357,83.786,88.571,89,88.714,90.857,89.143,89.857,91.714,93.286,93.143,96.214,89.357,89.214,89.714,89.714,90,90.071,90.357,90.714,91.214,86.071,86.143,86.143,86.357,86.429,93.429,93.429,93.429,93.429,86.5,86.5,86.571,86.571,86.429,91.429,91,91.214,91.143,90.714,90.071,89.143,88.857,86.286,93.286,93.286,93.214,93.357,95.143,95.071,95.5,95.286,95.286,95.714,89.571,90.357,88.929,90.786,89.286,86.286,86.214,86,79,79,79,79,79.071,79.071,85.571,79,80.786,81.5,82.929,85.929,86,86.071,93.071,93.071,93.071,86.071,86.143,86.143,86.429,93.143,90.714,90.143,89.286,86.429,86.286,86.286,86.286,86.286,86.286,93.286,93.214,93.214,93.214,90.929,88.643,89.214,90.071,92.929,93.071,93.071,93.071,86.143,86.143,86.214,86.214,86.214,86.214,81.5,85.071,84.429,84.071,82.643,82.5,79.286,79.143,84,84.143,82.429,79.214,79.143,79.071,79.071,80.143,80.786,81.143,82.286,76.214,78.929,76.643,78.571,78.214,78.429,80.571,79,79,84.214,84,82.143,81.714,79.5,85.643,86.071,81.5,81.643,81.857,83.286,84.357,86,86.071,83.214,81.286,78.929,79.286,80.071,80.571,80.5,87,86.286,86.214,86.214,79.214,79.214,79.214,83.857,84.929,88.786,83.357,84.929,77.5,77.643,78.143,78.857,78.929,78.929,81.214,79.143,79.071,75.5,76.143,75.786,79.214,79.286,86.214,84.357,83.571,82.714,80.357,79.357,84,82.929,82.571,85.5,83.357,83.929,86,79,79.143,80.786,80.714,81.143,83.286,83.5,83,82.571,82.714,83.357,86,86.143,86.214,92.929,92.643,92.5,90,87.714,87.857,81.643,82.214,85.786,86.071,84.429,79.071,78.929,72.714,72.5,73,72.929,76.286,79.071,79.143,86,86,85.929,79,80.643,85.929,86.071,92.214,92.786,88.429,88.143,87,86.214,86.214,86.214,79.214,79.214,86.143,86.143,86.143,86.143,86.143,86.143,90.214,90.929,85.071,85.071,84.286,83.714,87.071,86.071,86,79,79,79,79,79,79,77.643,79.214,80,80.786,81.5,85.143,84.786,85.143,91.214,88.714,86.5,86.429,86.357,79.357,75.643,81,79.714,79.143,79.143,79.214,80.5,79,80.357,81.429,80.5,79.143,79.071,85.786,89.5,88.571,90.143,89.5,89.286,86.071,86.214,87.357,79.857,79.643,81.643,82,81.714,82,80.786,81.714,81.286,75.5,75.714,78.857,78.857,78.929,86,87.643,88.786,89.857,90.286,90.286,85.929,86,86.071,93.071,93.071,93.071,93.071,93.071,93.071,93.071,93.071,92.143,86.571,86.643,93.571,93.571,93.571,93.571,93.571,91.214,86.571,86.643,86.643,86.643,85.643,83.143,88.357,86.286,86.214,79.143,79.214,79.214,79.214,81.5,83.5,79.214,79.143,79.214,80.214,83.643,82.929,84.643,83.857,86.357,86.357,78.591,71.858,65.184,61.091,58.623,51.959,45.127,38.338,31.5,25.782,19.095,12.873,12.248]
```


### 95th Percentile Latency

```mermaid
---
config:
    xyChart:
        width: 1200
        height: 400
    themeVariables:
        xyChart:
            titleColor: "#222"
            backgroundColor: "#fff"
            xAxisLineColor: "#222"
            yAxisLineColor: "#222"
            plotColorPalette: "#FF7F0E,#1F77B4,#D62728,#2CA02C,#9467BD,#8C564B,#E377C2,#7F7F7F,#BCBD22,#17BECF"
---
xychart-beta
    title "operation_type=read"
    x-axis "Time, m" 0 --> 10
    y-axis "Latency, ms" 20 --> 36
    line [25.702,27.108,27.452,28.151,27.896,27.386,27.248,27.464,27.39,27.692,27.969,28.459,28.512,27.637,27.718,26.566,25.874,25.388,25.175,25.504,25.751,26.154,25.463,24.377,24.154,23.608,23.029,24.156,23.892,24.484,24.958,25.73,25.937,26.198,24.234,24.412,24.586,24.839,25.308,25.369,27.113,27.216,27.299,27.292,26.397,25.246,25.347,24.946,26.178,26.105,26.318,27.006,26.841,26.372,25.894,25.312,25.453,25.487,26.563,27.461,27.47,26.993,27.636,27.631,27.488,27.113,27.099,27.5,27.108,27.186,26.117,26.364,25.541,25.493,25.717,26.628,25.985,24.934,25.521,25.798,25.688,26.526,26.634,27.164,27.821,27.094,27.762,27.932,28.425,28.174,27.853,28.979,28.955,28.812,28.962,28.619,28.673,28.571,28.397,28.767,28.449,28.555,28.126,27.954,28.228,28.121,27.604,27.892,27.668,26.454,26.561,26.723,27.494,28.075,28.242,26.828,26.306,26.544,26.775,26.753,27.392,26.526,26.905,28.259,28.221,27.829,27.671,26.713,26.805,27.008,27.126,27.214,26.741,26.721,26.49,27.112,26.864,26.195,26.342,26.506,26.38,27.021,26.902,27.074,27.058,27.063,26.838,27.168,27.565,27.39,28.072,28.161,28.332,28.499,28.244,27.931,28.24,28.924,29.208,28.937,28.822,28.606,27.748,27.659,26.775,26.7,26.079,25.665,25.45,25.407,25.445,23.835,23.976,24.728,25.591,25.242,26.387,26.532,25.734,25.91,26.321,26.503,27.432,27.476,26.91,27.988,27.914,27.52,27.417,27.096,26.462,26.79,27.644,27.785,27.939,28.093,28.138,28.171,27.954,27.812,27.355,27.16,27.914,28.221,28.622,28.416,28.392,27.971,27.943,27.426,27.387,27.255,28.052,27.471,27.873,28.061,27.5,27.886,27.896,27.552,28.114,28.254,27.864,28.453,28.566,28.824,28.711,29.532,29.577,29.572,29.597,29.038,28.785,28.929,28.922,28.792,29.433,29.589,29.448,29.065,28.796,28.625,28.701,28.902,28.348,28.915,29.048,29.497,29.229,29.654,29.801,29.998,30.021,30.437,30.823,30.214,30.023,29.64,29.965,30.269,30.351,29.887,30.196,29.015,28.45,28.352,28.161,28.18,27.93,28.769,28.222,28.749,29.239,28.812,28.47,28.886,28.54,29.946,29.957,29.535,28.958,28.762,29.191,29.099,30.088,29.76,29.641,29.737,29.65,29.923,30.138,29.904,30.093,30.255,31.057,31.118,30.816,30.883,30.635,30.312,30.369,30.204,30.665,30.453,30.491,30.247,29.499,29.301,29.11,28.986,29.125,29.008,27.948,28.369,27.642,27.283,26.506,25.75,25.386,26.17,26.667,27.085,26.786,26.991,26.737,26.767,27.351,27.528,28.077,28.534,29.287,29.061,29.063,28.889,29.205,29.276,29.253,29.015,28.996,29.157,29.821,29.883,29.623,29.694,29.313,29.823,29.743,28.652,28.503,28.505,28.634,28.966,29.54,29.519,29.203,28.75,28.013,27.931,28.149,28.123,28.32,29.91,29.755,30.211,30.418,29.568,28.98,29.275,29.59,29.872,30.988,30.964,31.054,30.823,31.059,30.967,31.209,30.659,30.618,31.095,30.77,30.603,29.971,30.43,30.366,30.224,29.851,29.913,29.679,29.525,29.36,29.265,29.309,29.749,29.894,29.458,29.948,29.239,28.994,28.986,28.554,28.849,28.921,28.737,29.309,29.161,28.771,28.267,27.57,27.859,28.339,28.898,28.691,29.342,29.938,30.404,30.458,29.976,29.122,29.45,28.973,29.664,30.452,30.737,29.99,29.446,29.671,28.443,28.371,28.436,28.226,28.772,29.171,29.4,30.165,29.808,29.795,28.82,28.988,29.34,29.027,30.145,30.422,29.634,29.617,29.381,29.277,29.249,29.238,28.878,29.165,30.199,30.578,30.835,31.004,30.526,30.315,30.669,30.686,30.223,29.719,29.643,29.669,29.929,29.638,28.961,27.857,27.648,27.579,27.424,27.309,27.371,27.619,27.535,28.358,27.738,27.565,27.156,26.727,27.123,27.713,27.614,27.723,28.293,28.122,27.136,26.203,26.898,26.272,26.508,26.315,27.05,27.745,27.504,28.19,28.005,27.636,26.748,26.975,27.4,27.807,27.671,28.732,28.755,29.145,28.658,28.325,28.635,27.861,27.635,28.108,28.472,28.581,28.759,28.752,28.751,28.537,27.5,27.732,28.21,28.365,28.567,29.246,29.674,29.825,29.853,29.788,30.405,29.88,29.933,29.507,30.045,29.806,30.205,30.063,29.875,29.689,29.819,30.016,30.035,29.42,29.142,30.167,29.932,30.266,30.67,30.843,29.968,29.524,29.064,29.391,28.915,28.456,28.379,29.377,29.154,28.977,28.535,27.935,28.006,27.436,27.623,27.836,27.772,27.41,27.567,27.733,28.327,27.928,27.697,27.581,28.08,28.525,28.39,28.455,28.597,29.045,29.635,29.594,29.722,29.513,28.384,28.222,28.795,28.655,31.908]
```


```mermaid
---
config:
    xyChart:
        width: 1200
        height: 400
    themeVariables:
        xyChart:
            titleColor: "#222"
            backgroundColor: "#fff"
            xAxisLineColor: "#222"
            yAxisLineColor: "#222"
            plotColorPalette: "#FF7F0E,#1F77B4,#D62728,#2CA02C,#9467BD,#8C564B,#E377C2,#7F7F7F,#BCBD22,#17BECF"
---
xychart-beta
    title "operation_type=write"
    x-axis "Time, m" 0 --> 10
    y-axis "Latency, ms" 17 --> 41
    line [21.397,20.842,21.907,21.595,23.313,23.739,23.794,23.848,22.933,26.099,25.462,26.53,27.059,26.788,26.158,27.667,27.611,28.214,28.082,29.317,30.299,30.424,29.821,29.259,29.259,29,29.263,28.241,29.494,29.494,30.803,30.17,29.843,28.195,26.129,26.401,26.438,26.197,26.506,26.158,28.473,29.484,28.665,27.921,25.764,25.074,26.458,25.764,28.049,27.939,28.948,29.19,28.665,27.902,24.5,24.023,24.157,24.833,25.62,28.097,27.724,28.859,26.487,27.648,27.406,25.574,25.281,27.139,27.957,29.207,29.674,28.464,27.939,26.197,23.397,21.257,20.386,19.946,19.928,20.823,21.734,23,23,20.825,22.192,21.286,22.591,24.521,26.808,28.548,29.976,29.314,29.207,27.939,28.464,27.101,27.387,26.656,26.981,28.682,28.396,27.212,26.57,25.829,25.199,28.464,28.278,30.023,29.732,29.547,28.065,29.82,28.014,27.622,27.722,27.88,27.274,27.88,26.958,23.667,26.596,25.19,27.253,29.207,30.548,28.965,29.9,29.674,30.451,30.177,30.723,30.516,31.417,31.588,28.81,27.611,26.788,24.5,23.463,23.728,23.816,23.358,20.2,19.966,19.977,21.31,21.429,21.922,26.942,27.994,28.788,29.345,30.63,31.463,30.989,31.073,30.95,32.273,31.94,31.969,31.172,31.571,30.03,30.518,29.342,28.805,26.514,26.45,25.826,25.559,27.92,26.785,28.746,29.93,29.805,28.75,30.581,30.548,30.807,29.804,30.169,30,31.1,32.3,32.866,31.94,31.541,30.121,30.871,31.208,28.416,30.897,30.698,32.882,32.985,33.218,33.545,33.382,32.062,31.684,31.29,31.601,32.046,31.4,31.121,27.554,27.012,25.04,24.5,22.087,20.318,19.98,24.424,25.942,25.02,23.833,21.455,23,23.729,25.357,27.849,27.727,27.957,31.351,31.156,30.5,28.393,28.903,29.773,29.98,30.765,30.879,30.468,30.452,28.096,28.895,30.606,29.884,30.845,31.417,30.95,30.699,30.5,31.152,31.51,32.351,32.509,32.677,33.099,32.923,32.936,31.932,32.134,31.472,31.56,31.738,32.266,31.691,32.014,31.618,32.059,31.335,31.381,30.931,30.012,30.25,30.25,31.55,32.521,32.721,32.351,31.952,30.099,28.5,27.722,28.241,27.957,28.984,29.207,28.522,27.406,26.197,24.875,24.071,23.92,24.6,24.76,26.266,27.313,27.63,30.088,29.409,29.409,29.172,29.16,28.489,29.176,29.609,30.905,31.385,31.811,32.647,33.601,33.712,32.392,32.069,31.541,31.156,30.321,32.422,31.712,31.712,30.983,30.995,31.722,31.218,29.82,29.618,31.5,31.533,31.625,32.505,32.861,33.026,33.38,31.951,31.951,31.3,30.5,30.414,30.8,30.717,28.75,28.54,29.375,28.388,29.667,29.111,29.263,30.012,30.129,30.826,31.452,30.5,29.886,29.563,29.917,30.328,28.833,28.512,29.663,28.911,28.5,29.224,28.689,28.662,28.75,28.654,28.715,29.614,29.384,29,29.64,31.141,29.19,28.673,29.463,30.429,31.065,31.226,30.778,31.574,31.801,31.713,32.277,33.752,34.077,33.908,34.932,35.423,34.752,34.335,34.145,35.056,35.532,35.659,36.463,35.744,35.238,34.33,33.917,33.929,33.976,33.2,33.724,34.092,33.528,31.574,31.447,30.37,28.731,28.962,29.56,29.074,28.963,28.137,28.081,27.364,26.568,23.277,22.214,25.26,23.783,24.5,23.98,25.981,26.019,26.375,25.44,24.375,21.893,24.231,26.904,28.983,29.621,28.435,30.727,30.169,32.893,32.379,32.983,33.137,32.703,34.171,35.197,34.431,33.445,33.242,34.173,33.743,32.104,31.015,29.271,28.898,27.964,29.7,32.074,32.422,29.657,30.196,31.517,31.91,31.91,32.265,32.771,33.404,33.25,32.932,32.797,31.582,29.927,28.675,29.558,30.339,29.691,29,28.171,28.731,26.634,26.958,25.603,25.957,28.825,29.713,29.759,28.525,29.625,28.458,28.019,29.209,28.951,28.586,30.255,29.375,29.85,29.922,27.307,27.618,28.26,29.481,29.976,30.657,31.217,30.25,30.378,30.116,30.112,30.674,29.994,28.554,29.571,28.372,28.489,28.642,26.852,25.9,24.136,23.793,23.381,23.452,24.787,27.342,29.073,30.867,30.041,30.5,30.602,30.665,30.61,30.518,30.966,31.583,31.94,31.777,31.706,31.281,32.119,31.381,31.565,31.937,31.729,31.355,31.721,31.898,32.243,31.54,30.778,30.578,31.561,31.745,30.863,31.434,31.286,31.286,31.462,31.286,30.922,31.035,31.636,32.162,32.995,32.505,31.266,31.226,31.629,32.906,32.92,32.688,32.672,33.197,32.851,32.536,32.465,32.303,31.326,33.365,32.985,31.314,31.071,28.056,28.524,27.884,28.411,28.112,28.979,29.318,28.095,28.411,28.79,25.432,25.516,29,27.071,32.237,31.318,36.227]
```

File: .github/workflows/slo.yml [hunk 1]
```diff
@@ -1,70 +1,98 @@
...
+        required: false
+      slo_workload_write_max_rps:
+        default: '100'
+        required: false
 
 jobs:
-  test-slo:
-    concurrency:
-      group: slo-${{ github.ref }}
+  ydb-slo-action-init:
```
[the remark this answers is not in the document]
Comment (polRk) [reply .github/workflows/slo.yml:25]: It doesn't matter. FYI https://github.com/ydb-platform/ydb-python-sdk/pull/513/files#diff-d292416fe8295dcfec82cb876d6b34a3552a9b29bd0b1c75058af766d89748d6R10

File: tests/slo/src/runner.py [hunk 2]
```diff
@@ -91,13 +91,13 @@ def run_slo(args, driver, tb_name):
     logger.info("Max ID: %s", max_id)
 
     metrics = Metrics(args.prom_pgw)
-    if SDK_SERVICE_NAME == "sync-python-table":
+    if SDK_SERVICE_NAME == "py-sync-table":
```
Comment (vgvoleg) [inline tests/slo/src/runner.py:94]: let's remove prefix `py-*`, all tests will be `py`

File: tests/slo/src/runner.py [hunk 3]
```diff
@@ -121,7 +121,6 @@ def run_from_args(args):
     driver_config = ydb.DriverConfig(
         args.endpoint,
         database=args.db,
-        credentials=ydb.credentials_from_env_variables(),
```
Comment (vgvoleg) [inline tests/slo/src/runner.py:124]: why have you deleted this? previous SLO was also with anonymous credentials

Comment (polRk) [reply tests/slo/src/runner.py:124]: Because it failed

Comment (vgvoleg) [reply tests/slo/src/runner.py:124]: Everything was fine with previous SLO. 
Probably there's something wrong with new YDB cluster? It is not good to hide the issue

**Review by vgvoleg — CHANGES_REQUESTED** [review]

**Review by vgvoleg — APPROVED** [review]

**Pull request merged**